cvat/apps/dataset_manager/tests/_test_formats.py

```diff
@@ -397,3 +401,96 @@ def load_dataset(src):
                     self.assertEqual(len(dataset), task["size"])
                 self._test_export(check, task, format_name, save_images=False)
 
+class FrameMatchingTest(_DbTestBase):
+    def _generate_task_images(self, paths):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=5416079)